src/analysis/raster/qgsrastercalcnode.h

```diff
@@ -42,7 +44,8 @@ class ANALYSIS_EXPORT QgsRasterCalcNode
       tOperator = 1,
       tNumber,
       tRasterRef,
-      tMatrix
+      tMatrix,
+      tFuncttion
```

Comment:
```suggestion
      tFunction
```

Reply:
yes, unable to write defined-by-me type correctly